src/Type/Accessory/AccessoryNumericStringType.php

```diff
@@ -312,6 +312,11 @@ public function isLowercaseString(): TrinaryLogic
 		return TrinaryLogic::createMaybe();
 	}
 
+	public function isUppercaseString(): TrinaryLogic
+	{
+		return TrinaryLogic::createMaybe();
```

Comment:
numeric string might be a lowercase-string because of the scientific "e" notation. But can a numeric string ever be an uppercase string?

Reply:
`1e100` and `1E100` are both numeric strings, so both LC/UC are allowed

and php prints scientific notation in UC by default - https://3v4l.org/Tg497 - so if `float` is casted into `string`, `numeric-string&uppercase-string` is the best narrowed type

Reply:
@ondrejmirtes @mvorisek Is this comment resolved, or do I need to do something here?

Reply:
Resolving this for now; please unresolve it as necessary.